Assets/Scripts/Models/Tile.cs

```diff
@@ -41,6 +41,7 @@ public Tile(int x, int y, int z)
         Characters = new List<Character>();
         MovementModifier = 1;
         Utilities = new Dictionary<string, Utility>();
+        ReservedAsWorkSpotBy = new HashSet<Furniture>();
```

Comment:
I got it now. ```IsReservedWorkSpot``` is only used when validating the furniture, but not when validating the Work Spot. So ```ReservedAsWorkSpotBy``` can be greater than 1.